makefiles/arch/native.inc.mk

```diff
@@ -0,0 +1,224 @@
+NATIVEINCLUDES += -DNATIVE_INCLUDES
+NATIVEINCLUDES += -I$(RIOTBASE)/core/lib/include/
+NATIVEINCLUDES += -I$(RIOTBASE)/core/include/
+NATIVEINCLUDES += -I$(RIOTBASE)/sys/include/
+NATIVEINCLUDES += -I$(RIOTBASE)/drivers/include/
+
+ifeq ($(OS),Darwin)
+  DEBUGGER ?= lldb
+else
+  DEBUGGER ?= gdb
+endif
+
+# only use pyterm wrapper if term target is requested
+ifeq (,$(filter term,$(MAKECMDGOALS)))
+  RIOT_TERMINAL ?= native
+else
+  RIOT_TERMINAL ?= pyterm
+endif
+
+RESET ?= $(RIOTBOARD)/native/dist/reset.sh
+FLASHER ?=
+FLASHFILE ?= $(ELFFILE)
+
+ifeq (pyterm,$(RIOT_TERMINAL))
+  TERMFLAGS += -ps $(FLASHFILE)
+  PROCARGS := --process-args
+else
+  TERMPROG ?= $(FLASHFILE)
+endif
+
+ZEP_IP ?= [::1]
+ZEP_PORT_BASE ?= 17754
+ifeq (1,$(USE_ZEP))
+ifeq (pyterm,$(RIOT_TERMINAL))
+  TERMFLAGS += --process-args '-z $(ZEP_IP):$(ZEP_PORT_BASE)'
+else
+  TERMFLAGS += -z $(ZEP_IP):$(ZEP_PORT_BASE)
+endif
+endif
+ifneq (,$(ZEP_MAC))
+ifeq (pyterm,$(RIOT_TERMINAL))
+  TERMFLAGS += --process-args '\-\-eui64=$(ZEP_MAC)'
+else
+  TERMFLAGS += --eui64=$(ZEP_MAC)
+endif
+endif
+
+export VALGRIND ?= valgrind
+export CGANNOTATE ?= cg_annotate
+export GPROF ?= gprof
+
+# basic cflags:
+CFLAGS += -Wall -Wextra -pedantic $(CFLAGS_DBG) $(CFLAGS_OPT)
+CFLAGS += -U_FORTIFY_SOURCE
+CFLAGS_DBG ?= -g3
+
+ifneq (,$(filter backtrace,$(USEMODULE)))
+  $(warning module backtrace is used, do not omit frame pointers)
+  CFLAGS_OPT ?= -Og -fno-omit-frame-pointer
+else
+  CFLAGS_OPT ?= -Og
+endif
+
+# default std set to gnu11 if not overwritten by user
+ifeq (,$(filter -std=%, $(CFLAGS)))
+  CFLAGS += -std=gnu11
+endif
+
+ifneq (,$(filter arch_64bit,$(FEATURES_PROVIDED)))
```

Comment:
CI doesn't like it if you do that, you might have to resort to a variable after all - or just move these two lines to the board

Reply:
Since we only have two boards, it would be easy to move it to the boards. On the other hand, I think it makes more sense to put the `CFLAGS` and `LINKFLAGS` in one place, rather than scattering them in different files.

Reply:
I could imagine having more specialized `native` based boards in the future too.